ext/pdo/php_pdo_driver.h

```diff
@@ -615,16 +615,12 @@ struct _pdo_stmt_t {
 			zval ctor_args;            /* freed */
 			zend_fcall_info fci;
 			zend_fcall_info_cache fcc;
-			zval retval;
 			zend_class_entry *ce;
 		} cls;
 		struct {
-			zval fetch_args;           /* freed */
+			zval dummy;           /* freed */
```

Comment:
Can this not be removed just like `retval`?

Reply:
No, this is needed for alignment reasons, follow-up is to rearrange more stuff to remove it.

Reply:
Maybe just add a short sentence comment so that future people won't attempt to remove this?